attest/eventlog.go

```diff
@@ -257,18 +258,19 @@ func (a *AKPublic) validateQuote(quote Quote, pcrs []PCR, nonce []byte) error {
 
 	sigHash := a.Hash.New()
 	sigHash.Write(quote.Quote)
+	digest := sigHash.Sum(nil)
 
 	switch pub := a.Public.(type) {
 	case *rsa.PublicKey:
-		if sig.RSA == nil {
-			return fmt.Errorf("rsa public key provided for ec signature")
-		}
```

Comment:
Why are we dropping this? Really we should end up with a counterpart for the ECC side to make sure we have an ECC signature for an ECC key.